app/assets/stylesheets/_colours.scss

```diff
@@ -0,0 +1,78 @@
+$black:         #000;
```

Comment:
Might be worth pulling in the compiled SCSS rather than the compiled CSS? We will refactor these colours in a coming sprint, so you will very much want to use the UI-Kit’s where possible. :S